checker-qual/src/main/java/org/checkerframework/checker/mustcallonelements/qual/OwningArray.java

```diff
@@ -0,0 +1,18 @@
+package org.checkerframework.checker.mustcallonelements.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.checker.mustcall.qual.Owning;
+
+/**
+ * Annotation indicating that ownership should be transferred to the annotated array for the
+ * purposes of Must Call checking. In that sense, the annotation ports the semantics of
+ * {@code @}{@link Owning} to arrays. The Must Call obligations of the annotated array mean that the
+ * listed methods must be called on each element of the array. When assigning to a This is a
+ * declaration annotation rather than a type annotation.
```

Comment:
What does "when assigning to a This" mean? Maybe there is a missing word in this sentence?

Based on the annotation declaration, I think this annotation is _always_ a declaration annotation, no?